ocp4/profiles/coreos-ncp.profile

```diff
@@ -189,16 +189,13 @@ selections:
     ### Install Required Packages
     - package_sssd-ipa_installed
     - package_aide_installed
-    #- package_dnf-automatic_installed
```

Comment:
Needed for auto apply security patches

Reply:
@shawndwells this is a brand new rule entirely as you cannot use yum/dnf.